test/jest/roktManager.spec.ts

```diff
@@ -1055,7 +1098,8 @@ describe('RoktManager', () => {
             const kit: Partial<IRoktKit> = {
                 launcher: {
                     hashAttributes: jest.fn(),
-                    selectPlacements: jest.fn()
+                    selectPlacements: jest.fn(),
+                    use: jest.fn(),
```

Comment:
Can you create a test for `use`?  Something like "should call kit.use with the same argument passed to mParticle.Rokt.use"